[2025-02-11T18:01:27.282+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T18:01:27.334+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data manual__2025-02-11T18:01:21.771868+00:00 [queued]>
[2025-02-11T18:01:27.350+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data manual__2025-02-11T18:01:21.771868+00:00 [queued]>
[2025-02-11T18:01:27.351+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-11T18:01:27.393+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_kaggle_data> on 2025-02-11 18:01:21.771868+00:00
[2025-02-11T18:01:27.473+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=93) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T18:01:27.478+0000] {standard_task_runner.py:72} INFO - Started process 94 to run task
[2025-02-11T18:01:27.486+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_to_s3_pipeline', 'download_kaggle_data', 'manual__2025-02-11T18:01:21.771868+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmpzrw3vr1d']
[2025-02-11T18:01:27.498+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask download_kaggle_data
[2025-02-11T18:01:27.620+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data manual__2025-02-11T18:01:21.771868+00:00 [running]> on host 3aae20995624
[2025-02-11T18:01:27.851+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_to_s3_pipeline' AIRFLOW_CTX_TASK_ID='download_kaggle_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T18:01:21.771868+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-11T18:01:21.771868+00:00'
[2025-02-11T18:01:27.855+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-11T18:01:27.996+0000] {s3_data.py:20} INFO - Downloading dataset wyattowalsh/basketball to /opt/***/data
[2025-02-11T18:01:28.000+0000] {s3_data.py:23} INFO - Dataset downloaded successfully.
[2025-02-11T18:01:28.001+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-11T18:01:28.031+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-11T18:01:28.033+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=kaggle_to_s3_pipeline, task_id=download_kaggle_data, run_id=manual__2025-02-11T18:01:21.771868+00:00, execution_date=20250211T180121, start_date=20250211T180127, end_date=20250211T180128
[2025-02-11T18:01:28.281+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T18:01:28.495+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T18:01:28.513+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
